configuration/ingress/README.md

```diff
@@ -0,0 +1,27 @@
+# Ingress Configuration
+
+This section covers ingress configuration options for the Howso Platform.
+
+## Overview
+
+The Howso Platform uses ingress to manage external access to services in your Kubernetes cluster. Key configuration options include:
+
+## Ingress Class
+
+Many ingress controllers will only process ingresses with a specific `ingressClassName`.
+
+Specify this with the `ingress.ingressClassName` key in the Howso Platform values.  It will be used for all ingresses.
```

Comment:
Is it okay to leave this unset, if your ingress controller is configured as the default ingress controller in the cluster?

Reply:
Already merged but I'll add these points to an update PR